native/effektio-test/Cargo.toml

```
@@ -14,8 +14,8 @@ features = ["with-mocks"]
 
 [dependencies]
 anyhow = "1"
-env_logger = "0.9.0"
+env_logger = "0.8"
 futures = "0.3.17"
+log = "0.4"
 tempfile = "3.3.0"
-tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
-zenv = "0.8.0"
\ No newline at end of file
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
```

Missing final newline
```suggestion
tokio = { version = "1", features = ["rt-multi-thread", "macros"] }

```